src/structs/stark_config.cairo

```diff
@@ -18,3 +20,62 @@ struct StarkConfig {
     // Number of layers that use a verifier friendly hash in each commitment.
     n_verifier_friendly_commitment_layers: felt252,
 }
+
+#[derive(Drop, Serde)]
+struct TracesConfig {
+    original: TableCommitmentConfig,
+    interaction: TableCommitmentConfig,
+}
+
+#[derive(Drop, Serde)]
+struct FriConfig {
+    // Log2 of the size of the input layer to FRI.
+    log_input_size: felt252,
+    // Number of layers in the FRI. Inner + last layer.
+    n_layers: felt252,
+    // Array of size n_layers - 1, each entry is a configuration of a table commitment for the
+    // corresponding inner layer.
+    inner_layers: Array<felt252>,
+    // Array of size n_layers, each entry represents the FRI step size,
+    // i.e. the number of FRI-foldings between layer i and i+1.
+    fri_step_sizes: Array<felt252>,
+    log_last_layer_degree_bound: felt252,
+}
```

Comment:
I opt for FRI functionalities & structs to be in fri module.
I think it should be moved to contextually relevant module WDYT?

Reply:
I think you quite misunderstood the concept behind this struct.
It might be called `DeserializationFriendlyFriConfig` - when you take a deeper look you can see that there is a implementation of the `into` trait to the struct with the same name in the `fri` module.
It is because to support objects (original struct has an array of structs instead of felts) in arrays I would need to modify parser which for our testing purposes is not a priority in my opinion.
The other way would be to modify the `fri` implementation to support this struct instead of the old one, but I've decided to do it the faster way for now and to implement `into` to the original struct.
This is only the "proxy" struct that helps in the deserialization from parser output to the `FriConfig` struct present in the `fri` module.
In my opinion such a bit "messy" struct used only for deserialization purposes for sure shouldn't be in the `fri` module which contains clean `fri` impl.

Reply:
Sure let's change its name to DeserializationFriendlyFriConfig then